### DISCUSSION TITLE
pc stats on desktop

### DISCUSSION THREAD

**Discussion by GreyCipher-sec** [i3/i3#5921] [Q&A] [answered]
I saw this version of garuda linux that uses i3vm and I would like to have pc stats and key bindings like in the picture, I also searched in the garuda i3 repository but couldn't find anything that would help, does anyone know what was used to do this?

![garuda-i3.jpg](https://github.com/i3/i3/assets/51584162/a806fe6b-b8f9-40a1-beb8-934c53d48f64)

**Comment by orestisfl** [comment, marked as answer]
You are likely referring to conky: https://github.com/i3/i3/discussions/5792
